actions/event-actions/changesWebAction.js

```diff
@@ -36,7 +36,7 @@ function main(params) {
           common.verifyTriggerAuth(triggerData, false)
             .then(() => EventProvider.validate(params))
             .catch(err => {
-              return reject(common.sendError(400, `Feed parameter validation failed`, err.message));
+              throw common.sendError(400, `Feed parameter validation failed`, err.message);
```

Comment:
Shouldn't this actually be what you had above? `Promise.reject(...)`?

Reply:
The existing code returns the result of the `reject` function which is a non-error value and causes the following `then` handlers to run (which is not what we want). By throwing the error, all the `then` handlers are skipped and the `catch` at the end can reject the returned promise with the error from the validation issue. Did you spot something else?

Reply:
If you return `Promise.reject` that should not happen. Javascript doesn't care (or even know) what error values are, as long as the promise is rejected underneath.